CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # PopupBridge Android Release Notes
 
+## unreleased
+* Support AppCompatActivity overload for PopupBridge::newInstance
+* Deprecate AppCompatActivity overload for PopupBridge::newInstance
```

Comment:
I do not understand the difference between these two lines...

Reply:
Should the first entry be support for FragmentActivity instead?

Reply:
Yeah this is a brain typo haha. Will fix 🙏